lib/perfdata/influxdbcommonwriter.cpp

```diff
@@ -109,20 +109,13 @@ void InfluxdbCommonWriter::Pause()
 	Log(LogDebug, GetReflectionType()->GetName())
 		<< "Flushing pending data buffers.";
 
-	Flush();
-
-	/* Work on the missing tasks. TODO: Find a way to cache them on disk. */
-	Log(LogDebug, GetReflectionType()->GetName())
-		<< "Joining existing WQ tasks.";
+	m_WorkQueue.Enqueue([this]() {
+		FlushWQ();
+	}, PriorityHigh);
```

Comment:
`PriorityHigh`? Are you sure? (Have a look how `RedisConnection#Sync()` is built.)